packages/cli/configuration/src/generators-yml/addGenerator.ts

```diff
@@ -67,7 +33,13 @@ export async function addGenerator({
                 name: normalizedGeneratorName,
                 ...invocation,
                 // Fall back to the hardcoded version if a "latest" does not yet exist
-                version: (await getLatestGeneratorVersion(normalizedGeneratorName, context)) ?? invocation.version
+                version:
+                    (await getLatestGeneratorVersion({
+                        cliVersion,
+                        generatorName: normalizedGeneratorName,
+                        context,
+                        includeRc: false
+                    })) ?? invocation.version
```

Comment:
thoughts on getting rid of `invocation.version` ?

Reply:
https://linear.app/buildwithfern/issue/FER-3131/remove-generatorinvocationsts for future

Reply:
chatted offline, but it's needed/updated while we migrate! so right now this would only really be expected to work for Python and Ruby SDKs, since Java, TS, etc aren't migrated and registered